maptowny-plugin/src/main/java/me/silverwolfg11/maptowny/managers/TownInfoManager.java

```diff
@@ -145,6 +138,17 @@ private void registerReplacements() {
                 t-> t.hasNation() ? TownyAPI.getInstance().getTownNationOrNull(t).getName() : ""
         );
 
+        registerParenthesesReplacement("nationstatus",
```

Comment:
minor comment: looks like the new code has slight formatting differences; would prefer if it matched the formatting of the old code, but no biggie